test/fuzz/fuzzer.ml

```diff
@@ -4,6 +4,11 @@ let timeout_count = ref 0
 
 let global_count = ref 0
 
+let write_module filename m = 
+  let oc = open_out filename in
+  Fmt.pf (Format.formatter_of_out_channel oc) "%a@." Owi.Text.pp_modul m;
+  close_out oc
```

Comment:
Can you use `Bos.OS.File.writef` instead ? You have an [example here](https://github.com/OCamlPro/owi/blob/8100737e7dc19982a5646875ec6a98ed849908d3/src/cmd/cmd_fmt.ml#L20C19-L20C37). It forces you to handle error properly (I just realized `open_out` is hidden by the `prelude` library in the whole code-base but not in the fuzzer...).